File: packages/player/android/src/main/kotlin/br/com/suamusica/player/PlayerState.kt
package br.com.suamusica.player

enum class PlayerState {
    IDLE,
    BUFFERING,
    PLAYING,
    PAUSED,
    STOPPED,
    COMPLETED,
    ERROR,
    SEEK_END,
    BUFFER_EMPTY,
    ITEM_TRANSITION,
    STATE_READY,
    STATE_ENDED,
}

File: packages/player/ios/Classes/PlayerState.swift
//
//  PlayerState.swift
//  smplayer
//
//  Created by Lucas Tonussi on 26/09/24.
//

import Foundation

enum PlayerState: Int {
    case idle = 0
    case buffering
    case playing
    case paused
    case stopped
    case completed
    case error
    case seekEnd
    case bufferEmpty
    case itemTransition
    case stateReady
    case stateEnded
}
